package.json

```diff
@@ -84,7 +84,7 @@
     "c8": "^7.12.0",
     "cookie": "^0.5.0",
     "dotenv": "^16.0.3",
-    "esbuild": "^0.17.5",
```

Comment:
It looks like we are updating from `0.17.5`, not `0.17.6`.

Reply:
I just updated the title of the issue.

Reply:
Yes. There's a typo in the PR title but not in the actual commit.